src/CommandingOfficers/CommanderStrong.java

```diff
@@ -125,50 +139,56 @@ protected void perform(MapMaster gameMap)
+    private static final int MOBILIZE_DEFENSE_BUFF = 10;
 
-    COModifier damageMod = null;
+    CODamageModifier damageMod = null;
+    CODefenseModifier defenseMod = null;
 
     MobilizeAbility(Commander commander)
     {
       super(commander, MOBILIZE_NAME, MOBILIZE_COST);
 
       damageMod = new CODamageModifier(MOBILIZE_BUFF);
+      defenseMod = new CODefenseModifier(MOBILIZE_DEFENSE_BUFF);
+      AIFlags = PHASE_TURN_START | PHASE_TURN_END;
     }
 
     @Override
     protected void perform(MapMaster gameMap)
     {
+      // Grant the base firepower/defense bonus.
       myCommander.addCOModifier(damageMod);
+      myCommander.addCOModifier(defenseMod);
 
-      // Make all foot-soldiers buildable from all buildings.
+      // Make inf/mechs buildable from all buildings.
       UnitModel infModel = myCommander.getUnitModel(UnitModel.UnitEnum.INFANTRY);
       UnitModel mechModel = myCommander.getUnitModel(UnitModel.UnitEnum.MECH);
-      UnitProductionModifier upm = new UnitProductionModifier(TerrainType.CITY, infModel);
+      UnitProductionModifier upm = new UnitProductionModifier(TerrainType.AIRPORT, infModel);
+      upm.addProductionPair(TerrainType.SEAPORT, infModel);
       upm.addProductionPair(TerrainType.CITY, mechModel);
-      upm.addProductionPair(TerrainType.AIRPORT, mechModel);
-      upm.addProductionPair(TerrainType.SEAPORT, mechModel);
-      upm.addProductionPair(TerrainType.HEADQUARTERS, infModel);
+      upm.addProductionPair(TerrainType.CITY, infModel);
       upm.addProductionPair(TerrainType.HEADQUARTERS, mechModel);
+      upm.addProductionPair(TerrainType.HEADQUARTERS, infModel);
       myCommander.addCOModifier(upm);
 
-      // Grant foot-soldiers and transports two (2) additional movement power.
+      // Grant a global +2 movement buff.
       COMovementModifier moveMod = new COMovementModifier(2);
-
-      // Use the high-capacity APC since we are swapping that model out for this turn.
-      moveMod.addApplicableUnitModel(highCapacityUnitModels.get(UnitModel.UnitEnum.APC));
-      moveMod.addApplicableUnitModel(myCommander.getUnitModel(UnitModel.UnitEnum.INFANTRY));
-      moveMod.addApplicableUnitModel(myCommander.getUnitModel(UnitModel.UnitEnum.MECH));
+      for( UnitModel model : myCommander.unitModels )
+      {
+        moveMod.addApplicableUnitModel(model);
+      }
       myCommander.addCOModifier(moveMod);
 
-      // Grant all transports extra cargo space, and TODO: let foot-soldiers hitch a ride on ground vehicles.
-      UnitRemodelModifier urm = new UnitRemodelModifier();
-      for( UnitModel.UnitEnum type : highCapacityUnitModels.keySet() )
+      // Lastly, all troops are refreshed and able to move again.
+      for( Unit unit : myCommander.units )
       {
-        urm.addUnitRemodel(myCommander.getUnitModel(type), highCapacityUnitModels.get(type));
+        if( unit.model.chassis == UnitModel.ChassisEnum.TROOP )
+        {
+          unit.isTurnOver = false;
+        }
```

Comment:
I guess we should consider adding an event for this so we can animate it, but that's probably best to leave for the future.